index.html

```diff
@@ -103,98 +121,115 @@ <h2 class="ptitle t2">My Recent Works</h2>
             src="./icons/mainproject.png"
             alt="My Latest Project"
           />
-          <h2 class="ptitle t1">Multi-Post Stories</h2>
```

Comment:
- [ ] The My Recent Works title should be in the same line with this vertical line see :arrow_down: 

- your design 

![888888888](https://user-images.githubusercontent.com/57562869/160007282-c729037c-a3b1-4456-9892-b7593818059f.png)


- Figma design 

![1222222222222](https://user-images.githubusercontent.com/57562869/160006945-c36af18c-ad0b-40fc-9081-31ae28b10501.png)

Reply:
I solved this issue as well, thank you!